internal/controllers/suite_test.go

```diff
@@ -77,7 +79,27 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
+	var operators operatorsv1alpha1.OperatorList
+	var bundleDeployments rukpakv1alpha1.BundleDeploymentList
+
+	Expect(cl.List(context.Background(), &operators)).To(Succeed())
+	Expect(cl.List(context.Background(), &bundleDeployments)).To(Succeed())
+
+	Expect(namesFromList(&operators)).To(BeEmpty(), "operators left in the cluster")
+	Expect(namesFromList(&bundleDeployments)).To(BeEmpty(), "bundle deployments left in the cluster")
```

Comment:
Would something like:
```suggestion
	Expect(operators.Items).To(BeEmpty(), "operators left in the cluster")
	Expect(bundleDeployments.Items).To(BeEmpty(), "bundle deployments left in the cluster")
```

suffice? Upside I see of your method is that it would show the names of the operator objects leftover, but I'm not sure what the `.Items` approach would output off the top of my head (maybe additional helpful info?)

Reply:
I had that before, and it shows a _huge_ output of the entire structure of the missing objects. To the point where the deluge of info made it seem less helpful.

With this PR getting the current number down to 0 and this test passing, any future failures should be pretty obvious where to look: in the code that was changed.